### DISCUSSION TITLE
Cannot loop over non-loopable variable: {{ seo:change_page_title }}

### DISCUSSION THREAD

**Discussion by lomotech** [studio1902/statamic-peak#212] [Q&A] [answered]
fresh install in statamic 3.3.6 using peak starting pack.
in php 8.1, valet, macOS

first load ok. then it will trigger error

```local.ERROR: Class "Spatie\LaravelIgnition\Recorders\LogRecorder\LogMessage" not found```

but when i take a look into laravel.log, there is pre-error before that

```local.DEBUG: Cannot loop over non-loopable variable: {{ seo:change_page_title }}```

did something missing in setting etc?

**Comment by robdekort** [comment, marked as answer]
Thanks for your report. I can't replicate the Log error. 

I can see the other debug notice when Globals -> SEO -> Change collection title is empty. It doesn't break anything though. 

You could wrap the code responsible (L12-L19) in `resources/views/_snippets.antlers.html` in  `{{ if seo:change_page_title }} {{ /if }}` to get rid of the debug message.

**Reply by lomotech** [reply]
thanks for your reply. i will try to recreate fresh project using same config to check if there is something im missing. right now im comment for a while to make sure the error is not triggered.

im mark it as solved for this time being.

**Reply by robdekort** [reply]
You could take out the {{ if }}. It's a bug in the new Runtime Parser and will be fixed in an upcoming release. The PR is already there I noticed.
